ci-operator/jobs/openshift/aws-vpce-operator/openshift-aws-vpce-operator-main-postsubmits.yaml

```
@@ -50,3 +50,63 @@ postsubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
+  - agent: kubernetes
+    cluster: build02
+    decorate: true
+    extra_refs:
+    - base_ref: main
+      org: openshift
+      repo: osde2e
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "false"
+    name: osde2e-rosa-stage-aws-vpce-operator-test-harness
```

We need a better name than "test-harness"
```suggestion
    name: osde2e-rosa-stage-aws-vpce-operator-post-submit
```